src/test/java/org/imsglobal/caliper/ri/events/HMHConstants.java

```diff
@@ -0,0 +1,15 @@
+package org.imsglobal.caliper.ri.events;
+
+public class HMHConstants {
+
+    public static final String BASE_URN = "urn:uuid:";
+    public static final String BASE_IRI = "https://www.hmhco.com/";
+    public static final String APP_NAME = "ReadingInventory";
+    public static final String STUDENT_USER_REF_ID = "0f4fedbe-2227-415f-8553-40731a627171";
+    public static final String STUDENT_NAME = "Casandra Rath";
+    public static final String ACTIVITY_REF_ID = "c050e852-5edb-4743-92d1-b53466de3a5f";
+    public static final String SCHOOL_REF_ID = "ab570d90-3791-4048-a5ee-759657ddccef";
+    public static final String DISTRICT_REF_ID = "ea827de8-0ce9-4fe4-a28c-eb388bd9eb92";
+
+    public static final String OBJECT_ID = "c35603f7a3434e2cb74ef14d60e70f42";
```

Comment:
Hi @krisiye I left this variable here because its used in multiple events, i think it should be replaced with unique ids per event - but left it here for you to look at....